proto/tpm_attestz.proto

```diff
@@ -56,14 +56,20 @@ message AttestResponse {
   string oiak_cert = 2;
 
   // Final observed unsigned PCR values {pcr_index -> pcr_value}.
-  map<int32, string> pcr_values = 3;
+  // protolint:disable MAX_LINE_LENGTH
+  // PCR bytes are formatted in accordance with section 10.3 of https://trustedcomputinggroup.org/wp-content/uploads/TCG_TPM2_r1p59_Part2_Structures_pub.pdf#page=111.
```

Comment:
it'd be nice if all of these comment lines with: 
  stuff here <url>
  
  were probably:
    stuff here
      <url>
      
      
maybe just a nit ! :) but possible easier ot read.